glustercli/cmd/volume-create.go

```diff
@@ -137,12 +137,20 @@ func volumeCreateCmdRun(cmd *cobra.Command, args []string) {
 		failure("Bricks not specified", nil, 1)
 	}
 
-	bricks, err := bricksAsUUID(args)
+	volname := flagCreateVolumeName
+	brickArgs := args
+	if volname == "" {
+		// old backward compatible style: glustercli volume create <volname> <bricks...>
+		volname = args[0]
```

Comment:
This can be done only when `args[0]` is not in brick format(`IP:Path` or `PeerID:Path`). Otherwise no way to create Volume with automatic name using CLI(Regular Volume usecase)

Reply:
I thought about it but I overlooked the [volume name restriction](https://github.com/gluster/glusterd2/pull/773) though - volume names cannot have `:`.

I can will the check, thanks :+1: